cg/models/orders/sample_base.py

```diff
@@ -99,9 +102,9 @@ class OrderSample(BaseModel):
     primer: str | None = None
     quantity: int | None = None
     reagent_label: str | None = None
-    reference_genome: constr(
-        max_length=Sample.reference_genome.property.columns[0].type.length
-    ) | None = None
+    reference_genome: (
+        constr(max_length=Sample.reference_genome.property.columns[0].type.length) | None
```

Comment:
regarding the sonar cloud issue, in the next file (`cg/models/orders/samples.py`) it seems to work with `regex=` instead of `pattern=`. Another solution would be migrate the model to pydantic 2